src/Observable.ts

```diff
@@ -96,9 +97,10 @@ export class Observable<T> implements CoreOperators<T>  {
       }
     } else {
       const next = <((x?) => void)> observerOrNext;
-      subscriber = Subscriber.create(next, error, complete);
+      subscriber = Subscriber.create(next, error, complete, start);
     }
 
+    subscriber.start(subscriber);
     subscriber.add(this._subscribe(subscriber));
```

Comment:
@blesh yes, but that happens after `_subscribe` runs. Don't you think it'd be more efficient if we didn't even execute the subscription if all we do is immediately throw it all away?